### DISCUSSION TITLE
Split tunnel for website

### DISCUSSION THREAD

**Discussion by benw8080** [Ysurac/openmptcprouter#2093] [Q&A]
Thanks Ysurac for a cool project,
I have a paid subscription service I am unable to access using OMPtcp Router, is it possible to split this off the VPN in any way? Its a streaming service - so assumption is they use a variable block of Ip addresses, is there a wildcard for the URl or site name possibly?  I can browse and sign in to the site but when I select a stream to view I get a nag warning asking to turn off the VPN. (sports site).
The same response happens on any device connected to the Omptcp router.
Thanks for your help!
Ben

**Comment by packlet** [comment]
Have you tried configuring OMR Bypass?    under Services - OMR-Bypass - Protocols and Services.  
https://ltehacks.com/viewtopic.php?t=1078&start=30#p9125
